### ISSUE TITLE
Magic tags not translating

### ISSUE THREAD

**Issue by ChrisHinde** [CalderaWP/Caldera-Forms#1524]
## What Version Of Caldera Forms, WordPress and PHP Are You Using?
WordPress Version: 4.7.4PHP Version: 7.0.17MySQL Version: 5.7.17Caldera Forms Version: 1.5.0.8WP_DEBUG:

## Does Your Issue Presist When You Disable All Other Plugins and Switch To The Default Theme?
Haven't tried switching to default theme (don't think it'll make a difference..) and disabled all plugins related to Caldera (except the main plugin), The other three shouldn't interfere (I'm not main admin, just tasked to get the form up and running)

## What Is The Unexpected Beahviour?
Magic tags doesn't work for variables and increment values. The tag is inserted by Caldera (eliminating spelling errors etc).
The fields of the stored entry still contains the tags:
![caldera](https://cloud.githubusercontent.com/assets/7074726/25259929/56944b62-2649-11e7-9c80-42017cd8a419.png)

## What PHP Errors Have You Logged While Reproducing This Bug?
None

## What JavaScript Errors Have You Seen While Reproducing This Bug?
None

**Comment by Shelob9** [comment]
@ChrisHinde Thanks for using Caldera Forms.

To be honest, I'm not sure what you are describing here. What does "Magic tags doesn't work for variables and increment values." mean? Can you explain what you did, what you were trying to accomplish, what you expected to happen, and what happened instead?

**Comment by ChrisHinde** [comment]
Maybe I wrote this too late in the night..
But the problem is that (some) "Magic tags" (AFAIK that is what they are called) doesn't translate/get turned into the value they represent. Not in the form and not in the processing of the form/final entry.

I hoped that the image would be helpful in showing the problem.
Here's a screenshot of the form, I expected that "{variable:logement_t}" would be replaced by the value of the variable logement_t.
![caform](https://cloud.githubusercontent.com/assets/7074726/25307472/c559c10e-27a1-11e7-97e0-c751b3ecb8f5.png)

Now, as I needed to get this to work, I skipped this route and used hooks and wrote my own solution instead.
